src/services/OParl/consultation.test.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  basicImportTest,
+  invalidUrlTest,
+  setupTestDataBase,
+} from '../../helpers';
+import { importConsultation } from './consultation';
+
+const helperFuncs = setupTestDataBase('test_consultation_import');
```

Comment:
setting up some db worked well without further doing, nice.

Reply:
i am very happy with the current db test setup, as it is indeed very easy to set up independent test databases like this.